docs/report/report.tex

```diff
@@ -45,6 +45,105 @@ \subsubsection{Stretch Goals}
 
 \section{Machbarkeitsstudie}
 
+\subsection{NFC Tag}
+\subsubsection{lesen}
+Um mit NFC Tags arbeiten zu können, müssen diese auch entschlüsselt bzw. gelesen werden können.
+Hierfür ist ein Hardware NFC-Reader notwendig, der die Daten ausliest und an die Box kommuniziert.
+Hierfür könnte \textit{evdev} genutzt werden, ein Kernel Modul von Linux. Mit \textit{node-evdev}\footnote{https://github.com/sdumetz/node-evdev} kann \textit{evdev} auch mit Node genutzt werden. Mit dem Fork\footnote{https://github.com/anbraten/node-evdev} von Anbraten wird auch Raspberry Pi und Typescript Unterstützung gestellt.
```

Comment:
Zweimal direkt hintereinander mit 'hierfür' anfangen?

Reply:
added suggestion in commit d8212f7f83371b83380e20a6080d67a105b4dd05